Contribute/content/how-to-write-quick-edits.md

```diff
@@ -0,0 +1,67 @@
+
+   :::image type="content" source="media/how-to-write-quick-edits/edit-article.png" alt-text="Screenshot of an Azure documentation article showing the edit pencil icon.":::
+
+   If the **Edit** button isn't present, it means the content isn't open to public contributions. Some pages are generated (for example, from inline documentation in code) and must be edited in the project they belong to.
+
+1. Select the pencil icon at the top of the GitHub file page. If the pencil icon is grayed out or doesn't display, you need to log in to your GitHub account.
+
+   :::image type="content" source="media/how-to-write-quick-edits/edit-icon.png" alt-text="Screenshot of the Azure article within GitHub,showing the edit pencil icon.":::
+
+    At the top of the article is the article's metadata. Metadata is applied to articles for reporting, discoverability via search, and driving aspects of the site experience. If you're making minor updates to a published article, you probably won't need to change the metadata.
+
+1. If it's your first time working in this repository, you'll be prompted to fork the repo before you propose changes. Select **Fork this repository** to continue.
+
+1. Edit the file in the web editor. Choose the **Preview** tab in the toolbar to check the formatting of your changes.
+
+1. When you're finished editing, select the **Commit changes** or **Propose changes** button, usually at the top-right of the screen.
+
+1. Enter a commit message. The commit message becomes the title of your PR and should be a brief summary of your changes (for example, "Fix spelling and grammar errors"). Optionally, add an **Extended description** to give more details about your changes. Select **Propose changes**:
+
+   :::image type="content" source="media/how-to-write-quick-edits/commit-changes.png" alt-text="Screenshot of the propose changes pop-up.":::
+
+1. Now that you've proposed and committed your changes, you need to ask the owners of the repository to "pull" your changes into their repository. This is done using a [pull request](https://docs.github.com/articles/using-pull-requests)(PR). When you select **Propose changes**, you'll see a page like this:
+
+   :::image type="content" source="media/how-to-write-quick-edits/create-pull-request.png" alt-text="Screenshot of the comparing changes screen.":::
+
+   Select **compare across forks** and make sure your branch configuration looks like that shown above. Your PR should propose changes from your fork and branch (represented by the two items on the right side of the arrow) into the MicrosoftDocs repo's fork and `main` branch (represented by the two items on the left side of the arrow). You may have to select the dropdown boxes to adjust the repo and branch names.
```

Comment:
I wrote it this way as I was going through the process as a new sample user under the current GitHub system. These steps reflect what I had to do. People are having to manually choose "compare across forks"; I think it's a bug, but it's how it works right now. A contributor even recently commented in our training module because the instructions _didn't_ follow this more manual way of doing it. 

Thanks for catching the MicrosoftDocs reference -- I'll change that.

Reply:
Actually, I just tried again and it's working as expected. I'll update the text!